app/controllers/campgrounds_controller.rb

```diff
@@ -7,8 +7,34 @@ def show
     @campground = Campground.find(params[:id])
   end
 
-  def index_campsites
-    show
-    @campsites = Campsite.where(campground_id: Campground.find(params[:id]))
+  def new
+  end
+
+  def create
+    @campground = Campground.create(campground_params)
+    redirect_to '/campgrounds'
+  end
+
+  def edit
+    @campground = Campground.find(params[:id])
+  end
+
+  def update
+    @campground = Campground.find(params[:id])
+    @campground.update(campground_params)
+    redirect_to "/campgrounds/#{@campground.id}"
+  end
+
+  def destroy
+    campground = Campground.find(params[:id])
+    campsites = campground.campsites
+    campsites.destroy_all
```

Comment:
Could consider adding `:dependent => :destroy` to your model relationship instead `campsites.destroy_all`